spam/01_spam_tutorial.py

```diff
@@ -30,8 +44,8 @@
 # %% [markdown]
 # We use a [YouTube comments dataset](https://www.kaggle.com/goneee/youtube-spam-classifiedcomments) that consists of YouTube comments from 5 videos. The task is to classify each comment as being
 #
-# * **`SPAM`**: irrelevant or inappropriate messages, or
-# * **`HAM`**: comments relevant to the video
+# * **`HAM`**: comments relevant to the video (even very simple ones), or
+# * **`SPAM`**: irrelevant (often trying to advertise something) or inappropriate messages
```

Comment:
im not sure whether all inappropriate messages are actually marked as spam...

Reply:
inappropriate is a catch-all for "rude" and "off-topic", so i think it works